eo-runtime/src/main/eo/org/eolang/txt/text.eo

```diff
@@ -23,13 +23,26 @@
 +package org.eolang.txt
 +home https://github.com/objectionary/eo
 +version 0.0.0
++alias org.eolang.collections.list
 +rt jvm org.eolang:eo-runtime:0.0.0
```

Comment:
@EugeneDar Could you please reorder meta according to alphabetically order?

Reply:
@Graur sure, I'll fix it.